OpenRA.Mods.Common/Traits/Crates/CrateAction.cs

```diff
@@ -27,6 +27,7 @@ public class CrateActionInfo : ITraitInfo
 		[PaletteReference] public readonly string Palette = "effect";
 
 		[Desc("Audio clip to play when the crate is collected.")]
+		[NotificationReferenceAttribute]
```

Comment:
This is not actual Notification reference, but direct sound filename. Bot sure why it is caled Notification.